doc/nrf/ug_zigbee_configuring_zboss_traces.rst

```diff
@@ -0,0 +1,151 @@
+************
+
+To collect and view the log files, you will need the following:
+
+* Software that allow for reading serial ports
+  For example, if you are using Windows you can use PuTTY. (See :ref:`putty` for more information.)
+
+* To find the correct COM port
+  One way do this is by setting the Kconfig option :kconfig:`CONFIG_USB_DEVICE_PRODUCT` (for example, :kconfig:`CONFIG_USB_DEVICE_PRODUCT="Zigbee sample"`)  to help identify the device in the system's :guilabel:`Device Manager`.
+
+* An additional USB cable if trace logs through USB are enabled.
+
+
+.. rst-class:: numbered-step
+
+Switch to ZBOSS libraries with compiled-in trace logs
+*****************************************************
+
+Set the Kconfig option :kconfig:`CONFIG_ZIGBEE_ENABLE_TRACES` to switch to ZBOSS libraries with compiled-in trace logs.
+
+The ZBOSS stack comes in a precompiled form and trace logs are not compiled-in by default.
+An additional set of ZBOSS libraries are available in nrfxlib, which does have trace logs compiled-in.
+
+
+.. rst-class:: numbered-step
+
+Select which ZBOSS trace logs to print
+**************************************
+
+1. Select which subsystems will receive logs by configuring the ZBOSS trace mask with the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_MASK`.
+   Trace masks can be created by adding masks of subsystems to receive the trace logs from.
+   For available subsystems, see :file:`nrfxlib/zboss/production/include/zb_trace.h`.
+
+#. Select the level of logs you want to receive. Configure ZBOSS trace level by selecting one of the following levels:
+
+   * Trace logs turned off - set :kconfig:`CONFIG_ZBOSS_TRACE_LOG_LEVEL_OFF`
+   * Error trace logs level - set :kconfig:`CONFIG_ZBOSS_TRACE_LOG_LEVEL_ERR`
+   * Warning trace logs level - set :kconfig:`CONFIG_ZBOSS_TRACE_LOG_LEVEL_WRN`
+   * Info trace logs level - set :kconfig:`CONFIG_ZBOSS_TRACE_LOG_LEVEL_INF`
+   * Debug trace logs level - set :kconfig:`CONFIG_ZBOSS_TRACE_LOG_LEVEL_DBG`
+
+See :ref:`zigbee_ug_logging_stack_logs` to read more about trace logs.
+
+
+
+.. rst-class:: numbered-step
+
+Configure how to print ZBOSS trace logs
+***************************************
+
+The :ref:`zigbee_osif_zboss_osif_serial` offers a few backends to choose from for printing ZBOSS trace logs.
+It is recommended to use the Zigbee serial logger, as it is the most efficient.
+To enable it, set the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_BINARY_LOGGING`.
+
+**Optional:** You can increase size of the ring buffer that temporarily stores the trace logs.
+   * To do this, set the Kconfig option :kconfig:`CONFIG_ZBOSS_TRACE_LOGGER_BUFFER_SIZE`.
+   * This can prevent losing some of the logs in demanding scenarios such as high network traffic, multiple devices being configured or joined, and so on.
+
+
+Trace logs using UART (default)
+===============================
+
+Trace logs using the UART are enabled by default.
+Below are more details on how to configure trace logs using the UART.
```

Comment:
```suggestion
To manually configure trace logs using the UART, complete the following steps:
```
You mean like "configure further" or "configure in case you disable it"?

Reply:
The latter, I believe

Reply:
I think this suggestion was not implemented... The point here was that we avoid using "below" and prefer "following", and usually use the standardized phrase "complete the following steps" to introduce procedures...